x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/loadingservice/LocalModel.java

```diff
@@ -29,7 +29,7 @@
 
 import static org.elasticsearch.xpack.core.ml.job.messages.Messages.INFERENCE_WARNING_ALL_FIELDS_MISSING;
 
-public class LocalModel<T extends InferenceConfig> implements Model<T> {
+public class LocalModel<T extends InferenceConfig> implements Model {
```

Comment:
```suggestion
public class LocalModel implements Model {
```
I think that bounded generic can be removed as well. It does not provide much information now that `infer` does not reference it.

Reply:
I think stuff in the model loading service + tests will have to change when this is removed.

Reply:
👍 I pushed that change